### ISSUE TITLE
In list of publications on Author aspect, clarify author versus author name string distinction

### ISSUE THREAD

**Issue by Daniel-Mietchen** [fnielsen/scholia#874]
Right now, it says "Authors" but does not link them, which is irritating.
We should perhaps write "Indentified authors" or some such, and link them.

**Comment by Daniel-Mietchen** [comment]
Example: https://tools.wmflabs.org/scholia/author/Q4693015 .

The query from there has to be modified a bit to give the author QID, as per [this](https://w.wiki/AU7):

```SPARQL

#defaultView:Table
SELECT
  (MIN(?dates) AS ?date)
  ?work ?workLabel
  (GROUP_CONCAT(DISTINCT ?type_label; separator=", ") AS ?type)
  (SAMPLE(?pages_) AS ?pages)
  ?venue ?venueLabel
  (GROUP_CONCAT(DISTINCT ?author ; separator=", ") AS ?authors)
WHERE {
  ?work wdt:P50 wd:Q4693015 .
  ?work wdt:P50 ?author .
  OPTIONAL {
    ?author rdfs:label ?author_label_ . FILTER (LANG(?author_label_) = 'en')
  }
  BIND(COALESCE(?author_label_, SUBSTR(STR(?author), 32)) AS ?author_label)
  OPTIONAL { ?work wdt:P31 ?type_ . ?type_ rdfs:label ?type_label . FILTER (LANG(?type_label) = 'en') }
  ?work wdt:P577 ?datetimes .
  BIND(xsd:date(?datetimes) AS ?dates)
  OPTIONAL { ?work wdt:P1104 ?pages_ }
  OPTIONAL { ?work wdt:P1433 ?venue }
  SERVICE wikibase:label { bd:serviceParam wikibase:language "en,da,de,es,fr,jp,no,ru,sv,zh". }  
}
GROUP BY ?work ?workLabel ?venue ?venueLabel
ORDER BY DESC(?date)    
```

However, this is missing the ?authorLabel, and I cannot figure out right now how to get that into the CONCAT phrase.

**Comment by Daniel-Mietchen** [comment]
Closed by #974  .
